ping360_sonar/SonarSettings settings
---
bool success